hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/addressbook/NodeCreateSuite.java

```
@@ -17,41 +17,344 @@
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNAUTHORIZED;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
+import com.google.protobuf.ByteString;
 import com.hedera.services.bdd.junit.HapiTest;
+import com.hedera.services.bdd.spec.keys.KeyShape;
+import com.hederahashgraph.api.proto.java.ServiceEndpoint;
+import java.util.Arrays;
+import java.util.List;
 import java.util.stream.Stream;
 import org.junit.jupiter.api.DynamicTest;
 import org.junit.jupiter.api.Tag;
 
 public class NodeCreateSuite {
+
+    public static final String ED_25519_KEY = "ed25519Alias";
+    public static List<ServiceEndpoint> GOSSIP_ENDPOINTS = Arrays.asList(
+            ServiceEndpoint.newBuilder().setDomainName("test.com").setPort(123).build(),
+            ServiceEndpoint.newBuilder().setDomainName("test2.com").setPort(123).build());
+    public static List<ServiceEndpoint> SERVICES_ENDPOINTS = Arrays.asList(ServiceEndpoint.newBuilder()
+            .setDomainName("service.com")
+            .setPort(234)
+            .build());
+    public static List<ServiceEndpoint> GOSSIP_ENDPOINTS_IPS =
+            Arrays.asList(endpointFor("192.168.1.200", 123), endpointFor("192.168.1.201", 123));
+    public static List<ServiceEndpoint> SERVICES_ENDPOINTS_IPS = Arrays.asList(endpointFor("192.168.1.205", 234));
+
+    /**
+     * This test is to check if the node creation fails during ingest when the admin key is invalid.
+     * @see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-869.md#specification">HIP-869</a>
+     */
+    @HapiTest
+    final Stream<DynamicTest> adminKeyIsInvalidOnIngest() {
+        return hapiTest(nodeCreate("nodeCreate")
+                .adminKeyName(NONSENSE_KEY)
+                .signedBy(GENESIS)
+                .hasPrecheck(KEY_REQUIRED)); // on ingest level before all the events on the handlers happens);
```

Should this throw `INVALID_ADMIN_KEY` or `BAD_ENCODING` instead ?